ang/civicase-features.ang.php

```diff
@@ -45,12 +45,18 @@ function set_currency_codes(&$options) {
 }
 
 /**
- * Exposes Case types that have quotations enabled to Angular.
+ * Exposes Case types that have features enabled to Angular.
  */
-function set_case_types_with_quotations_enabled(&$options) {
+function set_case_types_with_features_enabled(&$options) {
   $caseTypeCategoryFeatures = new CaseTypeCategoryFeatures();
-  $caseTypeCategories = $caseTypeCategoryFeatures->retrieveCaseInstanceWithEnabledFeatures(['quotations']);
-  $options['featureCaseTypes']['quotations'] = array_keys($caseTypeCategories);
+
+  array_map(function ($feature) use ($caseTypeCategoryFeatures, &$options) {
+    $caseTypeCategories = $caseTypeCategoryFeatures->retrieveCaseInstanceWithEnabledFeatures([$feature]);
+    $options['featureCaseTypes'][$feature] = array_keys($caseTypeCategories);
+  }, ['quotations', 'invoices']);
+
+  $caseTypeCategories = $caseTypeCategoryFeatures->retrieveCaseInstanceWithEnabledFeatures(['invoices']);
```

Comment:
I might be missing something here but this line and the one below seems redundant

Reply:
true, they are.